src/plugins/modifiers.ts

```diff
@@ -135,8 +139,48 @@ export class Modifiers {
                     const commandReplacer = directive.set !== undefined ? directive.set["command"] : undefined;
                     const commandDescriptionReplacer = directive.set !== undefined? directive.set["command-description"]: undefined;
                     const groupReplacer = directive.set !== undefined ? directive.set["group"] : undefined;
+                    const groupSplitter: any = directive.set !== undefined ? directive.set["split"] : undefined;
```

Comment:
>const groupSplitter: any = directive.set !== undefined ? directive.set["split"] : undefined; [](start = 20, length = 92)

Does it means split need to be set in directive? not auto?

Reply:
yes, just like in the description

Reply:
Is it possible that we support auto split?

---
In reply to: [411477973](https://github.com/Azure/autorest.az/pull/311#discussion_r411477973) [](ancestors = 411477973)

Reply:
+ Qiaoqiao for review. we'd better to split it automaticly

---
In reply to: [411842518](https://github.com/Azure/autorest.az/pull/311#discussion_r411842518) [](ancestors = 411842518,411477973)